docs/howtos/eviction-placement-disruption-budget.md

```diff
@@ -0,0 +1,356 @@
+# Using ClusterResourcePlacementEviction and ClusterResourcePlacementDisruptionBudget
+
+This how-to guide discusses how to create ClusterResourcePlacementEviction objects and ClusterResourcePlacementDisruptionBudget objects to evict resources from member clusters and protect resources on member clusters from voluntary disruption, respectively.
+
+## Evicting Resources from Member Clusters using ClusterResourcePlacementEviction
+
+The ClusterResourcePlacementEviction object is used to remove resources from a member cluster once the resources have already been propagated from the hub cluster.
+
+To successfully evict resources from a cluster, the user needs to specify:
+- The name of the ClusterResourcePlacement object which propagated resources to the target cluster
+- The name of the target cluster from which we need to evict resources.
+
+In this example, we will create a `ClusterResourcePlacement` object with PickAll placement policy to propagate resources to an existing MemberCluster, add a taint to the member cluster 
+resource and then create a ClusterResourcePlacementEviction object to evict resources from the MemberCluster.
+
+We will first create a namespace that we will propagate to the member cluster
+
+```
+kubectl create ns test-ns
+```
+
+Then we will apply a `ClusterResourcePlacement` with the following spec:
+
+```yaml
+spec:
+  resourceSelectors:
+    - group: ""
+      kind: Namespace
+      version: v1          
+      name: test-ns
+  policy:
+    placementType: PickAll
+```
+
+The CRP status after applying should look something like this:
```

Comment:
Just another nit: the full set of status here might be a bit redundant. Would using the `kubectl get` info columns be simpler?